### PR TITLE
(Receipt): Stock entry from depot with quantity difference

### LINKED ISSUE #3989: Stock Report : stock received versus stock sent report
BHIMA supports partial stock entry, so with this issue we must be able to show the difference between what a depot send and what another depot receive.

This report must have, for an entry from another depot :

- Inventory code
- Inventory description
- Sent quantity
- Received quantity
- Quantity difference
- Last entry date

### PR DESCRIPTION
**Pull request by mbayopanda**
This PR : 
- Add quantity received, quantity difference in the stock entry from depot receipt
- Fix description section

Close #3989

### CODE REVIEW

**Comment by kwilu** [comment]
Hello @mbayopanda, it seems like you have referenced #3989 in your pull request description, but you have not referenced them in your commit message description(s). When you reference an issue in a commit message, it automatically closes the corresponding issue when the commit is merged.

Please run `git commit --amend` in your command line client to amend your commit message description with `Fixes #3989.`.

An example of a correctly-formatted commit:
```
commit fabd5e450374c8dde65ec35f02140383940fe146
Author: kwilu
Date:   Sat Mar 18 13:42:40 2017 -0700

    pull requests: Check PR commits reference when issue is referenced.

    Fixes #51.
```

Thank you for your contributions to BHIMA!

**Comment by mbayopanda** [comment]
@jniles could i get a review of this PR please

File: server/controllers/stock/reports/common.js [hunk 1]
```diff
@@ -88,60 +88,67 @@ function formatFilters(qs) {
...
  */
-function getDepotMovement(documentUuid, enterprise, isExit) {
+async function getDepotMovement(documentUuid, enterprise, isExit) {
   const data = {};
   const isExitValue = isExit ? 1 : 0;
+  const isEntry = isExitValue === 0;
+
+  const lookupExitParameters = [1, Stock.flux.TO_OTHER_DEPOT, db.bid(documentUuid)];
+  const lookupEntryParameters = [0, Stock.flux.FROM_OTHER_DEPOT, db.bid(documentUuid)];
+  const parameters = isEntry ? lookupExitParameters.concat(lookupEntryParameters) : lookupExitParameters;
```
Comment (jniles) [inline server/controllers/stock/reports/common.js:98]: :+1:

**Review by jniles — APPROVED** [review]
Looks good to me!  

bors r+

**Pull request merged**